docs/reference_checklist.md

```diff
@@ -81,6 +80,11 @@ After implementing the interpreter:
        under test alphabetically based on the function name.
     1. Keep adding tests until the [ccov](https://github.com/openxla/stablehlo/blob/main/build_tools/github_actions/ci_build_stablehlo_code_coverage.sh)
        shows >= 90% coverage for the op.
+1. In [infer_stablehlo.mlir](https://github.com/openxla/stablehlo/blob/main/stablehlo/tests/infer_stablehlo.mlir):
+    1. Make sure all constraints related to shape inference tests are present
+       in this file, following the same naming guidelines noted above.
+    1. Move any shape inference tests from [ops_stablehlo.mlir](https://github.com/openxla/stablehlo/blob/main/stablehlo/tests/ops_stablehlo.mlir)
```

Comment:
Make sure that the tests in `ops_stableho.mlir` is portable to `infer_stablehlo.mlir`. Given that both the tests are using `--verify-diagnotics` this should not be a problem.

Reply:
They should be portable as `infer_stablehlo.mlir` has all the flags used in `ops_stablehlo.mlir`.